backend/Makefile

```diff
@@ -75,11 +75,11 @@ db/local/load-data:
 
 db/local/load-schema:
     # Imports the corpora_dev.sqlc schema (schema ONLY) into the corpora_test database
-	# Usage: make db/local/load-schema INFILE=<file>
+    # Usage: make db/local/load-schema INFILE=<file>
     $(eval DB_PW = $(shell aws secretsmanager get-secret-value --secret-id corpora/backend/test/database --region us-west-2 | jq -r '.SecretString | match(":([^:]*)@").captures[0].string'))
```

Comment:
make sure this is a tab not spaces. It showed up as space in my IDE and wouldn't run.